index.ts

```diff
@@ -51,11 +52,13 @@ if (!Array.isArray(repositories) || !Array.isArray(emails)) {
 
   console.log('Starting to commit to "./test-git-repo"');
 
+  process.chdir(TEST_REPO);
+
   for (const commit of sortedCommits) {
-    await git.commit("Another commit", {
-      "--allow-empty": null,
-      "--date": commit.date,
-    });
+    const gitCommand = `GIT_AUTHOR_DATE="${commit.date}" GIT_COMMITTER_DATE="${commit.date}" git commit --allow-empty --no-gpg-sign -m "Another commit"`;
```

Comment:
I initially reached for simple git worrying around arbitrary code exec from the git cli, but I'm happier with this, it's simpler and who is gonna write a command injection as a git commit date in their professional repos 😅

Reply:
Oh right. Indeed I don't think there much risk here, the only injectable part being `commit.date` which is the result of `Date.getTime()`.